Interview Questions

```diff
@@ -0,0 +1,78 @@
+# Interview questions for Junior Front-End Web Developers
+
+The purpose of this article is to familiarize fresh Front-End Web Developers with text book questions. If you are just getting into the developer world there is a good chance you will feel nervous when interviewing for a developer's positon. Because of this, some relatively basic questions could feel more difficult then how they actually are. A good counter for this scenario is to be properly prepared. Without further ado, here are a few questions that you could encounter:
+
+1)	Can you explain the meaning of “inline-block”?  
+
+
+
+2)	If you have and ul element with several li elements, how would you go about setting up these elements in a row by using HTML and CSS? 
+
+
+3)	What is the meaning of floats? How do they work? When would you use floats?
+
+
+4)	When would you use:  clear: both;?
+
+
+5)	What is Bootstrap, how does it work and why did you use it?
+
+
+6)	Explain the notion of responsive design and how it works? Use an example if you like.
+
+
+7)	What is DOM? And how would you explain it with your own words. 
```

Comment:
@alayek I don't think this is a interview question though :wink: 
> How is DOM represented in memory?

But I think the question is correct.
> Q: What is a DOM?
A: DOM is acronym of Document Object Model which is a standard specification which browser vendors must follow to provide an API for users to access and/or manipulate the browser behavior using scripting languages.